### PR TITLE
Graphics on notoifications and text (Rules not get saved on Next)

### PR DESCRIPTION
**Pull request by Devilla**

### CODE REVIEW

File: src/ducks/rules.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ const initialRules = {
   displayTime: 3,
   delayBetween: 3,
   displayPosition: 'bottom',
+  popupAnimation:'Fadeinout'
```
Comment (ShankyRana) [inline src/ducks/rules.js]: use lowercase

File: src/components/New/notification/display/right-view.js [hunk 2]
```diff
@@ -88,6 +96,23 @@ class RightView extends Component {
...
+          <Col md={6}>
+            <span className="mt-5">
+             Select Popup Notification Animation
+             </span>
+          </Col>
+      </Row>
+      <Row>
+             <Col md={6}>
+                    <FormControl value={popupAnimation} onChange={(e) => handleStateChange('popupAnimation', e.target.value)} componentClass="select" bsSize="sm" placeholder="select" >
+                      <option value="Fadeinout">Fade In/Out</option>
```
Comment (ShankyRana) [inline src/components/New/notification/display/right-view.js]: lowercase values

File: src/components/New/notification/display/index.js [hunk 3]
```diff
@@ -71,11 +73,13 @@ export class Display extends Component{
     let rule = this.state;
     rule['campaign']=this.props.campaign._id;
     if(rules) {
+      console.log(rules,"-----------------------------------------------------------------------------------")
```
Comment (ShankyRana) [inline src/components/New/notification/display/index.js]: remove logs

Comment (Devilla) [reply src/components/New/notification/display/index.js]: Please check branch : graphicsUpdate

**Pull request closed without merging**